### DISCUSSION TITLE
Hugo 基于 DocSearch 实现全文搜索

### DISCUSSION THREAD

**Discussion by Xuanwo** [Xuanwo/blog#231] [Announcements]
https://xuanwo.io/2021/02-docsearch-on-blog/ 

 An infrastructure engineer, focused on distributed storage system

**Comment by tianheg** [comment]
docsearch frontend **only works** with indices generated with the [docsearch crawler](https://github.com/algolia/docsearch-scraper). If you want to build a autocomplete dropdown you should use [autocomplete.js](https://xuanwo.io/2021/02-docsearch-on-blog/github.com/algolia/autocomplete.js) instead, which allow to build autocomplete dropdown on top of any indices.

这段里，autocomplete.js 的链接有问题

**Reply by Xuanwo** [reply]
Got it.

Fixed in https://github.com/Xuanwo/blog/commit/4d1c3b920bb78aa141d1bc4018a62d2242be40b8
